docs/branch_tracking.md

```diff
@@ -0,0 +1,53 @@
+# Branch tracking
+
+This requires basic knowledge on how Jujutsu's branches work[^1].
+
+As of December 2023 Jujutsu tracks branches by default, which is confusing 
+users coming from Git and Mercurial. To smoothen the transition branch tracking
+was introduced. 
+
+## Tracking a branch
+
+To track a branch permanently use `jj branch track <name-of-branch>`. It will 
+now be available in your local repository until you untrack it or it is 
```

Comment:
"imported as local branch" instead of "available in your local repository"?

I think "available" is misleading since remote branches can be specified by `<branch>@<remote>` syntax even if not imported to local branches.

Reply:
done. 

> I think "available" is misleading since remote branches can be specified by `<branch>@<remote>` syntax even if not imported to local branches.

Yes, your phrasing is definitely better with this feature in mind.